docs/styles.css

```
@@ -8,3 +8,20 @@ body {
   word-break: keep-all;
   word-wrap: break-word;
 }
+
+.navbar-github-link::before {
+  content: '';
+  width: 24px;
+  height: 24px;
+  display: flex;
+  transition: opacity 0.2s ease-in-out;
+  background: url(../static/img/github.svg) no-repeat center;
```

```suggestion
  background: url(../img/github.svg) no-repeat center;
```

I think it will work by [static-assets](https://docusaurus.io/docs/next/static-assets)

In docusaurus.config.js baseUrl is just "/" 
so the image /static/img/github.svg will be served ad /img/github.svg.

I think this will fix build error but if fail we have to find other solution!

---

Docusaurus will convert any file-based routings to a relative URI scheme. It was my mistake to oversee the `yarn prepare:docs` command!

---

oh! thank you I learned from you 🚀